exercises/anonymous-functions/anonymouns-functions6/main.go

```diff
@@ -0,0 +1,15 @@
+// anonymous functions2
+// Make me compile!
+
+// I AM NOT DONE
+package main
+
+import "fmt"
+
+func main() {
+
+	func(name string) {
```

Comment:
I could not understand the relation of the function with the hint you provided. Maybe you forgot to push the code?

The exercise name could be fixed: `anonymous-functions3`

Shouldn't it be `anonymous`? I think that is a typo

Reply:
Sorry, I did a mistake, forgot to rename some comments and folders. I'm gonna fix it and improve the anonymous-functions2 hint